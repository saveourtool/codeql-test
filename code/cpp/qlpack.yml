name: codeql-code-test
dependencies:
  codeql-queries-test: "*"
extractor: cpp
tests: .