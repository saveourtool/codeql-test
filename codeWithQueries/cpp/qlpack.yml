name: codeql-code-with-queries-test
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .