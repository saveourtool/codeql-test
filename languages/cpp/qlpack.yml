name: codeql-test-examples
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .