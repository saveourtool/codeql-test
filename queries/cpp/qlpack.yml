name: codeql-queries-test
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .