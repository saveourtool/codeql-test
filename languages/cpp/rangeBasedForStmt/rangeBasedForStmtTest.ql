/**
 * @name Test range based 'for' statement
 * @kind problem
 * @id cpp/rangeBasedForStmtTest
 */

import cpp

from RangeBasedForStmt rangeBasedForStmt
select rangeBasedForStmt, "This is range based 'for' statement."