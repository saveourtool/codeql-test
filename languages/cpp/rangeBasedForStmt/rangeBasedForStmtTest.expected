| rangeBasedForStmtCode.cpp:4:5:6:5 | for(...:...) ... | This is range based 'for' statement. |
| rangeBasedForStmtCode.cpp:13:5:15:5 | for(...:...) ... | This is range based 'for' statement. |
